compiler-cli/src/dependencies.rs

```diff
@@ -500,9 +500,11 @@ fn get_manifest<Telem: Telemetry>(
 
     let manifest = read_manifest_from_disc(paths)?;
 
+    let dependencies = get_all_dependencies(config)?;
```

Comment:
This detection is being done at the very edge of the system, when the CLI starts. It works for the CLI build tool as it always starts from scratch, but the problem will still exist for the language server which is a long-running process. We'll need to determine an approach that works in that case also.